packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -516,6 +518,13 @@ class CupertinoTextField extends StatefulWidget {
   /// Has no effect when [suffix] is null.
   final OverlayVisibilityMode suffixMode;
 
+  /// Controls the vertical aligment of the [suffix] and the [prefix] widget in relation to content.
```

Comment:
Might as well put the prefix first!

```suggestion
  /// Controls the vertical alignment of the [prefix] and the [suffix] widget in relation to content.
```